### PR TITLE
Feature/180148682 - Add single nucleus RNA-Seq experiment type

### PR DESCRIPTION
**Pull request by upendrakumbham**
The previous [PR](https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/224) has been closed because of conflicts due to the atlas-web-core submodule.

- I have created a new branch and merged develop to resolve conflicts and create a new PR.

-  This PR contains the implementation of the new experiment type:`SINGLE_NUCLEUS_RNASEQ_MRNA_BASELINE`.

- Testing: you can change the existing experiment type `SINGLE_CELL_RNASEQ_MRNA_BASELINE` to `SINGLE_NUCLEUS_RNASEQ_MRNA_BASELINE` and proceed. You should see the new experiment type all expected results like the previous experiment type.

- Associated PR: [atlas-web-core](https://github.com/ebi-gene-expression-group/atlas-web-core/pull/101)

### CODE REVIEW

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Can you please merge `develop` on top of this branch so that we get tests in Jenkins?

**Comment by haideriqbal** [comment]
Any updates on this @upendrakumbham ? I see CI is still failing?

**Comment by upendrakumbham** [comment]
> Any updates on this @upendrakumbham ? I see CI is still failing?

 I'm trying to import the 'E-ENAD-53' experiment into DB tables using `scxa_db` scripts. The fixture generation script works on DB data to generate fixtures if we have data in all the tables.

**Comment by pcm32** [comment]
I wonder why the github actions are not running here?

**Comment by pcm32** [comment]
ohh, I see, they were running until the previous to the last commit... maybe they come back up with the next commit or after the conflict resolution.

File: app/src/main/java/uk/ac/ebi/atlas/trader/ScxaExperimentRepository.java
Comment (alfonsomunozpomer) [inline app/src/main/java/uk/ac/ebi/atlas/trader/ScxaExperimentRepository.java]: This can (and should) be simplified to:
```java
if (!experimentDto.getExperimentType().isSingleCell())
```

And that will be always fine when we add new experiment types.

File: app/src/main/java/uk/ac/ebi/atlas/experimentimport/ScxaExperimentCrud.java
Comment (alfonsomunozpomer) [inline app/src/main/java/uk/ac/ebi/atlas/experimentimport/ScxaExperimentCrud.java]: Remove cache annotations. We’re not goin to add experiments via the web app. In time, also, the `ScxaExperimentCrud` class will be moved to the CLI.

File: app/src/test/java/uk/ac/ebi/atlas/experimentimport/ScxaExperimentCrudIT.java
Comment (alfonsomunozpomer) [inline app/src/test/java/uk/ac/ebi/atlas/experimentimport/ScxaExperimentCrudIT.java:81]: I like these tests! :smile:

File: app/src/test/java/uk/ac/ebi/atlas/experimentpage/ExperimentControllerWIT.java
Comment (alfonsomunozpomer) [inline app/src/test/java/uk/ac/ebi/atlas/experimentpage/ExperimentControllerWIT.java:83]: Good to keep the semantics of the test. It’s a shame that we can’t make more sophisticated assertions from `mockMvc`.

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Some comments to make the code cleaner. Very good job!

**Review by alfonsomunozpomer — DISMISSED** [review]
All good here too! :smiling_face_with_tear:

**Comment by upendrakumbham** [comment]
Hi @alfonsomunozpomer, I'm taking the liberty to merge this PR because after your approval again this PR got conflicts due to PR -229 merged to develop. So I resolved them again and the build got passed.  All good with atlas-web-core as well. Thanks.

**Pull request merged**
